### DISCUSSION TITLE
Do we need to specify sample rate to train a new model?

### DISCUSSION THREAD

**Discussion by kouohhashi** [archinetai/audio-diffusion-pytorch#32] [Q&A] [answered]
Hi,
Thank you for your great work.
I'm new here and have a question regarding frame rate.

In the document, it seems examples use 22050 but notebook examples seem to use 48000.
I want to experiment with speech datasets with frame rate of 16000.

Can I just input 16000 audio as input?
I mean do I need to set sample rate when creating model like ```model = AudioDiffusionModel(in_channels=1, frame_rate=16000)``` ?

```
# Can I just input 16000 raw audio as input x?
x = torch.randn(2, 1, 2 ** 18) # [batch, in_channels, samples], 2**18 ≈ 12s of audio at a frequency of 22050
loss = model(x)
loss.backward() # Do this many times
```

Thanks in advance!

**Comment by flavioschneider** [comment, marked as answer]
The model doesn't need to know about the sample rate since it's generic for 1D data. As long as inference and training have the same sample rate it's going to work fine. You might want to check out a few experiments examples in the trainer

**Reply by kouohhashi** [reply]
thanks I'll try it!
